content/en/platform/corda/4.8/open-source/node-docker-deployments.md

```diff
@@ -0,0 +1,194 @@
+
+{{< note >}}
+Requirements: A valid node.conf and a valid set of certificates - (signed by the CZ)
+
+{{< /note >}}
+In this example, the certificates are stored at `/home/user/cordaBase/certificates`, the node configuration is in `/home/user/cordaBase/config/node.conf` and the CorDapps to run are in `/path/to/cordapps`
+
+```shell
+docker run -ti \
+        --memory=2048m \
+        --cpus=2 \
+        -v /home/user/cordaBase/config:/etc/corda \
+        -v /home/user/cordaBase/certificates:/opt/corda/certificates \
+        -v /home/user/cordaBase/persistence:/opt/corda/persistence \
+        -v /home/user/cordaBase/logs:/opt/corda/logs \
+        -v /path/to/cordapps:/opt/corda/cordapps \
+        -p 10200:10200 \
+        -p 10201:10201 \
+        corda/corda-zulu-java1.8-4.8:latest
+```
+
+As the node runs within a container, several mount points are required:
+
+
+* CorDapps - CorDapps must be mounted at location `/opt/corda/cordapps`
+* Certificates - certificates must be mounted at location `/opt/corda/certificates`
+* Config - the node config must be mounted at location `/etc/corda/node.config`
+* Logging - all log files will be written to location `/opt/corda/logs`
+
+If using the H2 database:
+
+
+* Persistence - the folder to hold the H2 database files must be mounted at location `/opt/corda/persistence`
+
+{{< note >}}
+If there is no dataSourceProperties key in the node.conf, the Docker container overrides the url for H2 to point to the persistence directory by default so that the database can be accessed outside the container
+
+{{< /note >}}
+
+## Running a node connected to a Bootstrapped Network
+
+{{< note >}}
+Requirements: A valid node.conf, a valid set of certificates, and an existing network-parameters file
+
+{{< /note >}}
+In this example, we have previously generated a network-parameters file using the bootstrapper tool, which is stored at `/home/user/sharedFolder/network-parameters`
+
+```shell
+docker run -ti \
+        --memory=2048m \
+        --cpus=2 \
+        -v /home/user/cordaBase/config:/etc/corda \
+        -v /home/user/cordaBase/certificates:/opt/corda/certificates \
+        -v /home/user/cordaBase/persistence:/opt/corda/persistence \
+        -v /home/user/cordaBase/logs:/opt/corda/logs \
+        -v /home/TeamCityOutput/cordapps:/opt/corda/cordapps \
+        -v /home/user/sharedFolder/node-infos:/opt/corda/additional-node-infos \
+        -v /home/user/sharedFolder/network-parameters:/opt/corda/network-parameters \
+        -p 10200:10200 \
+        -p 10201:10201 \
+        corda/corda-zulu-java1.8-4.8:latest
+```
+
+There is a new mount `/home/user/sharedFolder/node-infos:/opt/corda/additional-node-infos` which is used to hold the `nodeInfo` of all the nodes within the network.
+As the node within the container starts up, it will place its own nodeInfo into this directory. This will allow other nodes also using this folder to see this new node.
+
```

Comment:
New when? Try to avoid 'new' as these docs should be around a long while, definitely outlasting any mount's newness.

Reply:
!! Thank you